frontend/src/components/widgets/Slider/Slider.tsx

```diff
@@ -55,7 +55,15 @@ class Slider extends React.PureComponent<Props, State> {
       DEBOUNCE_TIME_MS,
       this.setWidgetValueImmediately.bind(this)
     )
-    this.state = { value: this.props.element.get("default").toJS() }
+    this.state = { value: this.defaultValue() }
+  }
+
+  private defaultValue(): number[] {
```

Comment:
Why isn't this called `initialValue`? Just trying to be consistent.

Reply:
Good catch, fixed!